interface/interface.dm

```diff
@@ -72,11 +76,10 @@
 		var/url_params = "Reporting client version: [byond_version].[byond_build]\n\n[issue_template]"
```

Comment:
The legacy reporter is using ISSUE_TEMPLATE.md, you can make it use the new system and multiple alert/input popups.